src/dattobd.c

```diff
@@ -637,6 +635,7 @@ static int MAY_HOOK_SYSCALLS = 1;
 static unsigned long COW_MAX_MEMORY_DEFAULT = (300*1024*1024);
 static unsigned int COW_FALLOCATE_PERCENTAGE_DEFAULT = 10;
 static unsigned int MAX_SNAP_DEVICES = 24;
+static bool DATTO_DEBUG = 0;
```

Comment:
I think `bool` didn't exist if you go back far enough. change it to `int` if it didnt